CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.24.0 (December 13, 2021)
+
+IMPROVEMENTS
+
+* Added Get, Update, DeleteKeys, Associate, Disassociate, DisassociateAll, and ListAssociations for blockstorage QoS v3 via [GH-2283](https://github.com/gophercloud/gophercloud/pull/2283) and [GH-2284](https://github.com/gophercloud/gophercloud/pull/2284)

```

Comment:
I suggest you list all of the new methods individually, as previously done in past changelog entries.
```suggestion
* Added  `blockstorage/v3/qos.Get` [GH-2283](https://github.com/gophercloud/gophercloud/pull/2283)
* Added  `blockstorage/v3/qos.Update` [GH-2283](https://github.com/gophercloud/gophercloud/pull/2283)
* Added  `blockstorage/v3/qos.DeleteKeys` [GH-2283](https://github.com/gophercloud/gophercloud/pull/2283)
* Added `blockstorage/v3/qos.Associate` [GH-2284](https://github.com/gophercloud/gophercloud/pull/2284)
* Added `blockstorage/v3/qos.Disassociate` [GH-2284](https://github.com/gophercloud/gophercloud/pull/2284)
* Added `blockstorage/v3/qos.DisassociateAll` [GH-2284](https://github.com/gophercloud/gophercloud/pull/2284)
* Added `blockstorage/v3/qos.ListAssociations` [GH-2284](https://github.com/gophercloud/gophercloud/pull/2284)
```